### PR TITLE
use function provided by placmentrule for placement

### PR DESCRIPTION
**Pull request by kuanf**

### CODE REVIEW

File: pkg/controller/deployable/deployable_controller.go [hunk 1]
```diff
@@ -52,14 +54,127 @@ func Add(mgr manager.Manager) error {
...
+		if dpl.Spec.Placement == nil || dpl.Spec.Placement.PlacementRef == nil || dpl.Spec.Placement.PlacementRef.Name != obj.Meta.GetName() {
+			continue
+		}
+
+		annotations := dpl.GetAnnotations()
+		if annotations == nil || annotations[appv1alpha1.AnnotationManagedCluster] == "" {
+			continue
+		}
+
+		if annotations[appv1alpha1.AnnotationManagedCluster] != (types.NamespacedName{}).String() {
```
Comment (xiangjingli) [inline pkg/controller/deployable/deployable_controller.go]: Should it be 
```
if annotations[appv1alpha1.AnnotationManagedCluster] == (types.NamespacedName{}).String()  {
    continue
}
```

File: pkg/controller/deployable/deployable_controller.go [hunk 2]
```diff
@@ -52,14 +54,127 @@ func Add(mgr manager.Manager) error {
...
+				continue
+			}
+		}
+		// only reconcile it's own object
+		annotations := dpl.GetAnnotations()
+		if annotations == nil || annotations[appv1alpha1.AnnotationManagedCluster] == "" {
+			continue
+		}
+
+		if annotations[appv1alpha1.AnnotationManagedCluster] != (types.NamespacedName{}).String() {
```
Comment (xiangjingli) [inline pkg/controller/deployable/deployable_controller.go]: Same as line 101, Should it be 
```
if annotations[appv1alpha1.AnnotationManagedCluster] == (types.NamespacedName{}).String()  {
    continue
}
```

File: pkg/controller/deployable/deployable_controller.go (hunk 1 above)
Comment (kuanf) [reply pkg/controller/deployable/deployable_controller.go]: this one is correct.  `!= "\"` means it is a deployable for managed cluster. we shall skip (continue)

File: pkg/controller/deployable/deployable_controller.go (hunk 2 above)
Comment (kuanf) [reply pkg/controller/deployable/deployable_controller.go]: this one is correct. != "\" means it is a deployable for managed cluster. we shall skip (continue)

File: pkg/controller/deployable/deployable_controller.go (hunk 1 above)
Comment (xiangjingli) [reply pkg/controller/deployable/deployable_controller.go]: checked with Kuan, the comment is wrong  (types.NamespacedName{}).String() is "\", meaning the hub cluster.

File: pkg/controller/deployable/deployable_controller.go (hunk 2 above)
Comment (xiangjingli) [reply pkg/controller/deployable/deployable_controller.go]: see the reply in line 101. This is correct. (types.NamespacedName{}).String() = '\'

**Pull request merged**